packages/driver/src/cypress/utils.coffee

```diff
@@ -147,6 +147,19 @@ module.exports = {
     .join('\n\n')
     .value()
 
+
+  findParent: (el, fn) ->
+    prevEl = el
+    curEl = el.parentElement
+    retEl = null
+    while (curEl)
```

Comment:
generally i prefer using a function and tail recursion as opposed to a `while` statement because it's clearer and oftentimes its easier to refactor and additionally provides a "name" to indicate how it's traversing